package.json

```diff
@@ -102,7 +102,7 @@
         "@lit/react": "^1.0.4",
         "@open-wc/dev-server-hmr": "^0.2.0",
         "@open-wc/testing": "^4.0.0",
-        "@playwright/test": "^1.44.0",
+        "@playwright/test": "1.53.1",
```

Comment:
Removed carrot to lock version to the specific version requirement for web-test-runner so we will no longer have playwright browser install issues locally and CI